README.md

```diff
@@ -1,70 +1,84 @@
+    ├── notebooks          <- Jupyter notebooks.
     |    |
-    │    ├── poc           <- Proof of concept notebooks
+    │    ├── poc           <- Proof of concept notebooks.
     |    |
-    │    ├── eda           <- Exploratory data analysis
+    │    ├── eda           <- Exploratory data analysis.
     │    │
-    │    ├── modelling     <- Used for training and evaluation documentation
+    │    ├── modelling     <- Used for modelling exploration and evaluation.
     │    │
-    │    └── evaluation    <- Evaluation of model results
-    │
+    │    └── evaluation    <- Evaluation of model results.
     │
-    ├── models             <- Storage of model weights, nested by model architecture
+    ├── models             <- Storage of model weights, nested by model architecture.
     │    |
-    │    ├── ARCH_X        <- Directory for architecture X
+    │    ├── ARCH_X        <- Directory for architecture X.
     |        ├── ARCH_X_001.h5
     |        ├── ARCH_X_001.json
     |        │
-    |        └── ARCH_X_002  
-    │  
+    |        └── ARCH_X_002
+    │
     ├── src                 <- Source code to use in this project.
     |    |
-    │    ├── __init__.py    <- Makes src a Python module
+    │    ├── __init__.py    <- Makes src a Python module.
     |    |
-    │    ├── utils.py       <- General utility functions   
+    │    ├── utils.py       <- General utility functions.
+    |    |
+    │    ├── main.py        <- Main code for running an API/application.
+    │    │
+    │    ├── app            <- Code used in deployment.
+    │    │
+    │    ├── preparation    <- Data preparation. Retrieves data and puts in data/raw/, transforms to data/processed/ using src/processing/.
     │    │
-    │    ├── preparation    <- Data ingestion and retrieval. Retrieves data and puts in data/raw
+    │    ├── processing     <- Data transformation and pre-processing.
     │    │
-    │    ├── processing     <- Data transformation and pre-processing. Processes data from data/raw -> data/processed
+    │    ├── training       <- Code for training and evaluating models.
     │    │
-    │    └── modelling      <- Model classes, training and evaluation scripts
+    │    └── modelling      <- Model classes.
     │
-    ├── test                <- Code used for testing src/
+    ├── test                <- Code used for testing src/.
     │
-    │    
-    ├── requirements.txt    <- The requirements file for reproducing the analysis environment
+    ├── requirements.txt    <- The requirements file for reproducing the environment.
     │
-    ├── setup.py            <- Makes project pip installable (pip install -e .) so src can be imported
+    ├── setup.py            <- Makes project pip installable so src can be imported.
 
 ## How to work - phases
 
-#### POC phase
+### POC phase
 
 In this phase, you may use some kaggle dataset or a small collected dataset. Focus should be on displaying potential and not building a whole sophisticated system.
```

Comment:
förslag: ta bort "you may". låter preskriptivt. "In this phase, use some ..." tycker jag låter bättre

Reply:
Ska bli